### ISSUE TITLE
Doesn't work with the latest `scribejava` version (6.9.0)

### ISSUE THREAD

**Issue by boris-petrov** [boncey/Flickr4Java#362]
Doing:

```java
new Flickr(apiKey, sharedSecret, new REST());
```

When in the classpath is the latest `scribe` (6.9.0) instead of the one used by `Flickr4Java` (6.2.0) leads to:

```
java.lang.NoClassDefFoundError: com/github/scribejava/core/builder/api/BaseApi
```

**Comment by boncey** [comment]
I'll take a look - in the meantime can you stick to scribe 6.2.0?

**Comment by boris-petrov** [comment]
@boncey - thanks for the fast reply!

I can for a short while, but it's going to be an issue in the long run. I hope it's going to be a rather easy migration for you - most likely the changes will be small and not as big as [last time](https://github.com/boncey/Flickr4Java/pull/300).

**Comment by boncey** [comment]
This is fixed now in release 3.0.4. 👍
